converter/src/test/java/gov/cms/qpp/conversion/model/error/AllErrorsTest.java

```diff
@@ -53,4 +55,11 @@ public void testToString() {
 		assertWithMessage("Must contain formatted string")
 				.that(objectUnderTest.toString()).contains(error.toString());
 	}
+
+	@Test
+	public void testArgConstructor() {
+		List<Error> errors = new ArrayList<>();
+		errors.add(new Error());
+		new AllErrors(errors);
```

Comment:
I would call `AllErrors#getErrors` afterward to verify that it is the same as the constructor argument `errors`.